From 094be9242970026746871f6c15d886dc098aadc8 Mon Sep 17 00:00:00 2001
From: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
Date: Wed, 31 Jul 2013 14:40:47 +0900
Subject: [PATCH 0827/1083] Revert "drm: rcar-du: Add LVDS interface function"

This reverts commit 497c2bb5aaa145b2426528506091af5c2400552d.

Conflicts:
	drivers/gpu/drm/rcar-du/rcar_du_crtc.c
	drivers/gpu/drm/rcar-du/rcar_du_drv.c
	drivers/gpu/drm/rcar-du/rcar_du_drv.h
---
 drivers/gpu/drm/rcar-du/rcar_du_regs.h | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_regs.h b/drivers/gpu/drm/rcar-du/rcar_du_regs.h
index 0dff14e..b11e98c 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_regs.h
+++ b/drivers/gpu/drm/rcar-du/rcar_du_regs.h
@@ -197,14 +197,6 @@
 #define DEFR6_MLOS1		(1 << 2)
 #define DEFR6_DEFAULT		(DEFR6_CODE | DEFR6_TCNE2)
 
-#define DEFR8			0x20020
-#define DEFR8_CODE		(0x7790 << 16)
-#define DEFR8_VSCS		(1 << 6)
-#define DEFR8_DRGBS_DU2		(2 << 4)
-#define DEFR8_DRGBS_DU1		(1 << 4)
-#define DEFR8_DRGBS_DU0		(0 << 4)
-#define DEFR8_DEFE8		(1 << 0)
-
 /* -----------------------------------------------------------------------------
  * R8A7790-only Control Registers
  */
@@ -518,15 +510,4 @@
 #define GCBCR			0x11098
 #define BCBCR			0x1109c
 
-/* -----------------------------------------------------------------------------
- * LVDS registers
- */
-
-#define LVDCR0		0x0000
-#define LVDCR1		0x0004
-#define LVDPLLCR	0x0008
-#define LVDCTRCR	0x000C
-#define LVDCHCR		0x0010
-
-
 #endif /* __RCAR_DU_REGS_H__ */
-- 
1.8.3.2

